configs/kind/kubeadm.conf

```diff
@@ -1,12 +1,12 @@
 apiVersion: kubeadm.k8s.io/v1beta3
 kind: ClusterConfiguration
-kubernetesVersion: v1.23.5
+kubernetesVersion: v1.25.3
 networking:
   podSubnet: 192.168.0.0/16
 scheduler:
   extraArgs: null
 ---
-apiVersion: kubelet.config.k8s.io/v1beta1
+apiVersion: kubelet.config.k8s.io/v1beta3
```

Comment:
Shouldn't it be `v1`?

Reply:
I'll rather drop the related commit, these config were related to fixing CI runner within the containers